### PR TITLE
Shorten format substitution reference names

### PR DESCRIPTION
**Pull request by deoren**
@rgerhards Since we're already using these references in other documents (e.g., #427), I'm going ahead and submitting this batch of changes now instead of waiting until the other docs have been updated.

refs rsyslog/rsyslog-doc#421

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -49,14 +49,9 @@
...
 
-.. |FormatsBasicOldName| replace:: ``sysklogd``
-.. |FormatsBasicNewName| replace:: ``basic``
-
-.. |FormatsAdvancedOldName| replace:: ``RainerScript``
-.. |FormatsAdvancedNewName| replace:: ``advanced``
-
-.. |FormatsObsoleteOldName| replace:: ``legacy``
-.. |FormatsObsoleteNewName| replace:: ``obsolete legacy``
+.. |FmtBasicNewName| replace:: ``basic``
```
Comment (rgerhards) [inline source/conf.py]: sorry for being insistent... do we need "New" inside the name? ```FmtBasicName``` would be good IMHO...

Comment (deoren) [reply source/conf.py]: No, we don't; good catch. I think I meant to do that based on your prior suggestion, but forgot about it when I made this quick change.

Provide via a separate commit or squash with the previous one?

**Comment by rgerhards** [comment]
squash ;-)

**Comment by deoren** [comment]
Corrected, squashed, changes pushed.

**Pull request merged**